src/components/search/curated/node/index.tsx

```diff
@@ -99,4 +87,70 @@ Node.js uses an event-driven, non-blocking I/O model that makes it lightweight a
   )
 }
 
+export const nodePageQuery = groq`
+*[_type == 'resource' && slug.current == "node-js-landing-page"][0]{
+  title,
+  description,
+  summary,
+  "ogImage": images[label == "og-image"][0].url,
+  "topicImage": images[label == "topic-image"][0].url,
+  "beginner": resources[slug.current == 'beginner'][0] {
+    title,
+    "name": meta,
+    resources[]-> {
+      byline,
+      title,
+      path,
+      slug,
+      image
+    }
+  },
+  "intermediate": resources[slug.current == 'intermediate'][0]{
+    title,
+    "name": meta,
+    resources[]-> {
+      byline,
+      title,
+      path,
+      slug,
+      image
+    }
+  },
+  "advanced": resources[slug.current == 'advanced'][0]{
+    title,
+    "name": meta,
+    resources[]-> {
+      byline,
+      title,
+      path,
+      slug,
+      image
+    }
+  },
+  "featureCourse": resources[slug.current == 'feature-course'][0] {
+    "title": resources[0]-> title,
+    "byline": resources[0]-> byline,
+    "path": resources[0]-> path,
+    "slug": resources[0]-> slug,
+    "image": resources[0]->image 
+  },
+  "apiAndPerformance": resources[slug.current == 'api-and-performance'][0] {
+    "course": resources[0]->{
+      byline,
+      title,
+      path,
+      "slug": slug.current,
+      image
+    },
+    "article": resources[1]{
+      byline,
+      title,
+      path,
+      "slug": slug.current,
+      image
+    }
+  }
+}
+`
```

Comment:
Page query. When building the document in Sanity, I used Sanity Vision to build this